draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,92 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>Note: The following algorithm does not yet have WG
+          consensus but is included here as something concrete for the
+          working group to discuss."</t>
+          <t>When an RTP packet is received by a transport and the
+          SRTP has been validated, that packet needs to be routed to
+          the correct RtpReceiver. For each transport, the following
+          steps MUST be followed to prepare to route packets:</t>
+
+          <t>
+            <list>
+              <t>Construct a table mapping MID to RtpReceiver for each
+              RtpReceiver configured to receive from this transport.
+              The MID of a given RtpReceiver is found in the a=mid
+              line of the m= section corresponding to that RtpReceiver
+              in remote local and remote descriptions.</t>
+              
+              <t>Construct a table mapping SSRC to RtpReceiver for
+              each RtpReceiver configured to receive from this
+              transport and for each SSRC that RtpReceiver is
+              configured to receive.  The SSRCs of a given RtpReceiver
```

Comment:
That's not the only place where the SSRCs can come from, and they're not required. I would say to populate the table using the supplied SSRCs, if present.